oneswap_helper.rb

```diff
@@ -1249,7 +1285,7 @@ def hybrid_downloader(vc_disks)
         puts 'Downloading disks from vCenter storage to local disk'
         vc_disks.each_with_index do |d, i|
             # download each disk to the work dir
-            remote_file = d[:backing][:fileName].split(' ')[1].gsub(/\.vmdk$/, '-flat.vmdk')
+            remote_file = d[:backing][:fileName].sub(/^\[.+?\] /, '').sub(/\.vmdk$/, '-flat.vmdk')
```

Comment:
probably wise to check for errors here?

Reply:
I added some checking for if nothing comes back from there, the download will also raise an error if the path ends up being incorrect.